index.d.ts

```diff
@@ -0,0 +1,49 @@
+declare module 'atomizer' {
+    export interface AtomizerConfig {
+        custom?: Record<string, string>;
```

Comment:
Custom values can be objects too: https://github.com/acss-io/atomizer/blob/master/tests/atomizer.js#L319-L326

Reply:
updated https://github.com/acss-io/atomizer/pull/370/commits/67373c462bf230f058ee1e33f50e003d98f417f2